tests/cli/test_cmds_validate.py

```diff
@@ -172,6 +173,52 @@ def test_marathon_validate_schema_list_hashes_good(
 
 @patch('paasta_tools.cli.cmds.validate.get_file_contents')
 @patch('sys.stdout', new_callable=StringIO)
+def test_marathon_validate_schema_healthcheck_cmd_has_cmd(
+    mock_stdout,
+    mock_get_file_contents
+):
+    marathon_content = """
+---
+main_worker:
+  cpus: 0.1
+  instances: 2
+  mem: 250
+  disk: 512
+  cmd: virtualenv_run/bin/python adindexer/adindex_worker.py
+  healthcheck_mode: cmd
+"""
+    mock_get_file_contents.return_value = marathon_content
+
```

Comment:
This looks a little strange with so many newlines inbetween the code, is this intentional?